dynamo/preprocessing/preprocessor_utils.py

```diff
@@ -66,7 +66,7 @@ def is_log1p_transformed_adata(adata: anndata.AnnData) -> bool:
     chosen_gene_indices = np.random.choice(adata.n_vars, 10)
     _has_log1p_transformed = not np.allclose(
         np.array(adata.X[:, chosen_gene_indices].sum(1)),
-        np.array(adata.layers["spliced"][:, chosen_gene_indices].sum(1)),
+        np.array(adata.layers["X_spliced"][:, chosen_gene_indices].sum(1)),
```

Comment:
we should use spliced here because this check whether adata.X is normalized and spliced store the original raw data

Reply:
Done.